sdk/core/core-client/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Updates package to work with the react native bundler. Browser APIs may still need to be pollyfilled for this package to run in react native. [PR #17783](https://github.com/Azure/azure-sdk-for-js/pull/17783)
+
```

Comment:
I am curious, is there a way to document those APIs with mappings to the appropriate polyfills?

Reply:
The pollyfill landscape in react native is kind of still like the wild west. I plan to have an example shims file in the samples that are included in a follow-up PR, but I don't want to recommend any specific 3rd party pollyfill at this time since I haven't properly vetted them beyond ensuring some seem to work.

Reply:
That is fair. In order to make it easier for react native customers to use the SDK, should we consider providing documentation of non-standard definitions that need to be polyfilled for every package?

Reply:
My plan was to document that in react-native samples (that live alongside our current react/electron samples.) Happy to include some documentation at that point as well similar to how we have documentation on bundling.

Reply:
Sounds great! Thanks!